keyboards/cosmo65/cosmo65.h

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022 BentoBox Studio
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see http://www.gnu.org/licenses/.
+*/
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
```

Comment:
This, and the cosmo65.c files can be removed, actually.

Reply:
For the "This," at the beginning of your sentence, are you still referring to changing all the LAYOUT_all to LAYOUT? Thanks